docs/config.rst

```diff
@@ -1,11 +1,606 @@
+        class_path: cumulusci.tasks.salesforce.Deploy
+        group: projectName
+        options:
+            path: unpackaged/config/reports
+
+.. note:: Be sure to include the value we retrieved for ``class_path``. It's also best practice to add a common ``group`` attribute to your custom tasks in order to make it easier to see the tasks that are specific to your project when using ``cci task list``.
+
+Congratulations! You created a new custom task in CumulusCI.
+
+For additional examples of custom tasks, visit the `task recipes<TODO>`_ section in the cookbook.
+
+If you want to `write a custom task in Python<TODO>`_ and make it available to other users in the project, add the task under the ``tasks`` section of the ``cumulusci.yml`` file:
+
+.. code-block:: yaml
+
+    tasks:
+        my_new_task:
+            description: Description of the task
+            class_path: tasks.task_file.MyNewTaskClassName
+            group: projectName
+
+The above assumes that your task's class is named ``MyNewTaskClassName`` and exists in the file ``tasks/task_file.py``.
+
+
+Use Variables for Task Options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When you want to reference a specific value within the ``cumulusci.yml`` file, use the ``$project_config`` variable.
+
+    Example: NPSP uses a variable for the project's namespace by setting a value of ``$project_config.project__package__namespace``. This variable is then referenced in the project's custom ``deploy_qa_config`` task where it passes as the value for the ``namespace_inject`` option.
+
+    .. note:: A double underscore ( ``__`` ) allows access to different levels of the ``cumulusci.yml`` file.
+
+.. code-block:: yaml
+
+    deploy_qa_config:
+            description: Deploys additional fields used for QA purposes only
+            class_path: cumulusci.tasks.salesforce.Deploy
+            group: Salesforce Metadata
+            options:
+                path: unpackaged/config/qa
+                namespace_inject: $project_config.project__package__namespace
+
+In this instance CumulusCI replaces the variable with the value currently located under project -> package -> namespace in the ``cumulusci.yml`` file. Here is the ``project`` section of NPSP's ``cumulusci.yml`` file:
+
+.. code-block:: yaml
+
+    project:
+        name: Cumulus
+        package:
+            name: Cumulus
+            name_managed: Nonprofit Success Pack
+            namespace: npsp
+            api_version: 48.0
+            install_class: STG_InstallScript
+            uninstall_class: STG_UninstallScript
+
+Currently under ``$project_config.project__package__namespace`` is the value ``npsp``.
+
+
+Reference Task Return Values
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Tasks can set an internal ``return_value`` on themselves while executing. This attribute allows one task in a flow to reference the ``return_value`` set on another task that executed prior to it.
+
+To reference a return value on a previous task use the following:
```

Comment:
```suggestion
To reference a return value on a previous task:
```
Simpler wording and we usually don't need "following." https://confluence.internal.salesforce.com/display/DOCTEAM/following